docs/_sass/_docs-display-component.scss

```diff
@@ -15,7 +15,7 @@
 
     > .fd-tile__content {
```

Comment:
I think those stylings should be even removed. They create false expectations

Reply:
This rule is applied only when the fd-tile is child of `.docs-component`.
The padding is just for the content to look nice and the width is to show the cases when we have very long text and truncation.
We want to be able to see this:
<img width="1451" alt="Screen Shot 2019-11-08 at 10 52 21 AM" src="https://user-images.githubusercontent.com/39598672/68490040-4b0e8400-0216-11ea-924f-e2722d9466ec.png">

Without the width we see this:
<img width="2197" alt="Screen Shot 2019-11-08 at 10 53 57 AM" src="https://user-images.githubusercontent.com/39598672/68490055-52359200-0216-11ea-9d58-cc36415f90af.png">